test/widgets/action_sheet_test.dart

```diff
@@ -616,4 +634,4 @@ void main() {
       check(mockSharePlus.sharedString).isNull();
     });
   });
-}
+}
```

Comment:
nit: looks like this is unintended?

Reply:
Bump. We should add an empty line to the end of the file.

Reply:
(Not an empty line — the problem is that every line should end with a newline, and this commit is editing the file's last line by removing its terminating newline. Previous discussion: https://github.com/zulip/zulip-flutter/pull/534#discussion_r1515353889 )